grammars/toml.cson

```diff
@@ -21,19 +21,50 @@
     'name': 'key.toml'
   }
   {
+    'begin': '"""'
+    'beginCaptures':
+      '0': 'name': 'string.dquote.triple.begin.toml'
+    'end': '"""'
+    'endCaptures':
+      '0': 'name': 'string.dquote.triple.end.toml'
+    'name': 'string.dquote.triple.toml'
```

Comment:
`string.quoted.single/double/triple.toml` for 'string', "string", """string""".
`punctuation.definition.string.begin/end.toml` for `'`, `"`, `"""`
See [naming conventions](http://manual.macromates.com/en/language_grammars#naming_conventions) (no info about about `punctuation`)